tests/integration_test.go

```diff
@@ -760,21 +766,23 @@ func TestError_ShouldOnlyTakeOfferOnce(t *testing.T) {
 		}
 
 		for status := range takerStatusCh {
-			t.Log("> XMRTaker got status:", status)
+			t.Log("> XMRTaker[2] got status:", status)
 			if status.IsOngoing() {
 				continue
 			}
 
 			if status != types.CompletedSuccess {
-				errCh <- fmt.Errorf("swap did not exit successfully: got %s", status)
+				errCh <- fmt.Errorf("2nd swap did not exit successfully: got %s", status)
 				return
 			}
 
-			t.Log("> XMRTaker exited successfully")
+			t.Log("> XMRTaker[1] exited successfully")
```

Comment:
```suggestion
			t.Log("> XMRTaker[2] exited successfully")
```

also meganit but i'd index them as 0 and 1 lol, or just by name